.github/gen-npm-pjt.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+set -eux
+
+VERSION=$(cat version)
+
+cat <<EOF > package.json
+{
```

Comment:
Why not storing this template in a file in a dedicated file? We would benefit from linter and it would be easier to edit, I guess.

Reply:
Sure. I'll let you decide what's best. Thanks. :relieved:

Reply:
@amimart Maybe you can put the file `package.json` under the `.github` directory (as it only relates to CI), that way it doesn't taint (too much) the project with `npm`?

Reply:
@ccamel I really love that idea thank you ! 🙏

Reply:
@ccamel I did the update in https://github.com/okp4/cosmos-faucet/pull/39/commits/1647f3c1ff7dfee17e94d6ae9080b91e49f9b14b but I couldn't find a solution to let the file in place and properly embed the schema.. I had to copy it in the right folder from the ci publish workflow before launching the npm publish.

It seems files can't be referenced through relative paths, If you have any idea..

Otherwise, being used only for publishing the schema I don't think it's a big deal..